game_eggs/steamcmd_servers/icarus/egg-icarus--dedicated.json

```diff
@@ -0,0 +1,75 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-12-09T14:55:27+01:00",
+    "name": "Icarus-Dedicated",
+    "author": "bolverblitz@ebg.pw",
+    "description": "Icarus is a survival game that with dedicated servers as a public beta",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "Staging": "ghcr.io\/parkervcp\/yolks:wine_staging",
+        "Latest": "ghcr.io\/parkervcp\/yolks:wine_latest"
+    },
+    "file_denylist": [],
+    "startup": "wine64 .\/Icarus\/Binaries\/Win64\/IcarusServer-Win64-Shipping.exe -Log -SteamServerName=${SERVER_NAME} -PORT=\"${SERVER_PORT}\" -QueryPort=\"${QUERY_PORT}\"",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"(Engine Initialization) Total time:\"\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n\r\n## License: MIT License\r\n## Image to install with is 'ghcr.io\/parkervcp\/installers:debian'\r\n\r\n# Required for Wine to install vc_redist\r\nENV WINEPREFIX=\/home\/steam\/vcredist\r\nENV WINEARCH=win32\r\nENV WINEPATH=\/\r\n\r\n# Install packages. Default packages below are skipped if using image noted above, thus speeding up the install process.\r\napt -y update\r\npt -y --no-install-recommends install curl lib32gcc-s1 ca-certificates\r\n\r\n# Download and install SteamCMD\r\nexport HOME=\/mnt\/server\r\ncd \/tmp\r\nmkdir -p $HOME\/steamcmd $HOME\/steamapps\r\ncurl -sSL -o steamcmd.tar.gz https:\/\/steamcdn-a.akamaihd.net\/client\/installer\/steamcmd_linux.tar.gz\r\ntar -xzvf steamcmd.tar.gz -C $HOME\/steamcmd\r\ncd $HOME\/steamcmd\r\n\r\n# SteamCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\n\r\n# Install game server using SteamCMD\r\n.\/steamcmd.sh +force_install_dir $HOME +login anonymous $( [[ \"${WINDOWS_INSTALL}\" == \"1\" ]] && printf %s '+@sSteamCmdForcePlatformType windows' ) +app_update 2089300 validate +quit\r\n\r\n# Set up 32 and 64 bit libraries\r\nmkdir -p $HOME\/.steam\/sdk{32,64}\r\ncp -v linux32\/steamclient.so $HOME\/.steam\/sdk32\/steamclient.so\r\ncp -v linux64\/steamclient.so $HOME\/.steam\/sdk64\/steamclient.so\r\n\r\nEOF",
```

Comment:
1. you use the installer image so the apt update and apt install on line 12 and 13 (even a `a` is missing on line 13)
2. wy is there just a random EOF at the end of the install script ?
3. where is the install done message that present in your example scripts?
4. this: `export HOME=/mnt/server` is missing on the line below `chown -R root:root /mnt` <https://github.com/parkervcp/eggs/blob/master/scripts/steamcmd_installer.sh>
5. wine is not even in the install image is this realy needed? : 
```bash
ENV WINEPREFIX=/home/steam/vcredist
ENV WINEARCH=win32
ENV WINEPATH=/
```